### PR TITLE
HAWQ-1355. Fix namespace check may occur multiple times in first query.

### PR DESCRIPTION
**Pull request by linwen**
Please review, thanks!

### CODE REVIEW

**Comment by paul-guo-** [comment]
I'm wondering whether rename *.ans.orca
<https://github.com/apache/incubator-hawq/pull/1146#diff-6d2d67ad13a2a11e3e525ec0fff5be11>
->
*.orca.ans and for *.ans.planner -> *.planner.ans. This helps potential
wildcard processing in the future.

The line number could change again with code change. Maybe you could use
the init_file mechanism to avoid some ans file change.

2017-02-27 11:40 GMT+08:00 Xiang Sheng <notifications@github.com>:

**Comment by zhangh43** [comment]
+1

File: src/test/feature/UDF/ans/function_basics.ranger.ans.planner [hunk 1]
```diff
@@ -1045,7 +1133,7 @@ SELECT outer(0);
 (1 row)
 
 SELECT outer(0) FROM foo;
-psql:/tmp/TestUDF_TestUDFBasics.sql:423: ERROR:  function inner(integer) does not exist
+psql:/tmp/TestUDF_TestUDFBasics.sql:423: ERROR:  function inner(integer) does not exist  (seg2 localhost:40000 pid=63812)
```
Comment (zhangh43) [inline src/test/feature/UDF/ans/function_basics.ranger.ans.planner]: (seg2 localhost:40000 pid=63812) should be handled by init_file.

**Comment by zhangh43** [comment]
@paul-guo-  This PR already use init_file, please check incubator-hawq/src/test/feature/lib/global_init_file for details.

**Pull request closed without merging**